uint32 camera_id
string view_mode
---
sensor_msgs/Image image
